mul_arm64.h

```diff
@@ -1,4 +1,9 @@
-#define mul(c0,c1,c2,c3,c4,c5,c6,c7) \
+#define EMPTY
+
+#define RELOAD \
+	MOVD ·p2+0(SB), R5
+
+#define mul(c0,c1,c2,c3,c4,c5,c6,c7,t0,reset) \
```

Comment:
This might make this a bit more clear.
```suggestion
#define ResetR5 \
	MOVD ·p2+0(SB), R5

#define mul(c0,c1,c2,c3,c4,c5,c6,c7,maybeR5,maybeResetR5) \
```